Description: Upstream changes introduced in version 5.22.1-1
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 maxima (5.22.1-1) unstable; urgency=low
 .
   * New upstream release
   * Bug fix: "Please reduce recommends from maxima to maxima-emacs to
     suggests", thanks to Petter Reinholdtsen (Closes: #574171).
   * Bug fix: "Please enable alternative pdf-viewer / postscript-viewer",
     thanks to Andreas Tille (Closes: #590785).
 .
 The person named in the Author field signed this changelog entry.
Author: Camm Maguire <camm@debian.org>
Bug-Debian: http://bugs.debian.org/574171
Bug-Debian: http://bugs.debian.org/590785

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- maxima-5.22.1.orig/interfaces/emacs/misc/bookmode.el
+++ maxima-5.22.1/interfaces/emacs/misc/bookmode.el
@@ -680,7 +680,7 @@ file")
 		
 		  (and tem
 		       (list (concat "View "(nth 1 tem))
-			     'call-process  "ghostview" nil nil nil
+			     'call-process  "see" nil nil nil
 			     (expand-file-name(nth 1 tem))
 			     ))
 		  (list "Cancel")
@@ -705,7 +705,7 @@ file")
 			 (concat
 			  (cond ((looking-at "%PS") "")
 				(t "gzip -dc | "))
-			  "ghostview -")
+			  "see -")
 			 
 	)))
 	 
--- maxima-5.22.1.orig/interfaces/xmaxima/Tkmaxima/Constants.tcl
+++ maxima-5.22.1/interfaces/xmaxima/Tkmaxima/Constants.tcl
@@ -148,9 +148,9 @@ set maxima_priv(urlHandlers) {
     image/gif  netmath
     image/png  netmath
     image/jpeg netmath
-    application/postscript "ghostview -safer %s"
-    application/pdf "acroread %s"
-    application/x-dvi "xdvi %s"
+    application/postscript "see %s"
+    application/pdf "see %s"
+    application/x-dvi "see %s"
 }
 set maxima_priv(imagecounter) 0
 
--- maxima-5.22.1.orig/src/init-cl.lisp
+++ maxima-5.22.1/src/init-cl.lisp
@@ -521,8 +521,9 @@ When one changes, the other does too."
     ;; Autoload for Maxima documantation index file
     (let ((subdir-bit (if (null *maxima-lang-subdir*) "." *maxima-lang-subdir*)))
       ;; Assign AUTOLOAD property instead of binding a function (the result of AUTOF).
-      (setf (get 'cl-info::cause-maxima-index-to-load 'autoload)
-	    (combine-path *maxima-infodir* subdir-bit "maxima-index.lisp")))))
+      (unless (get 'cl-info::cause-maxima-index-to-load 'autoload)
+	(setf (get 'cl-info::cause-maxima-index-to-load 'autoload)
+	    (combine-path *maxima-infodir* subdir-bit "maxima-index.lisp"))))))
 
 (defun get-dirs (path)
   #+(or :clisp :sbcl :ecl :openmcl)
--- maxima-5.22.1.orig/lisp-utils/defsystem.lisp
+++ maxima-5.22.1/lisp-utils/defsystem.lisp
@@ -4671,6 +4671,7 @@ the system definition, if provided."
 			  source-pname
 			  :output-file
 			  output-file
+			  #+gcl :system-p #+gcl t
 			  #+(or :cmu :scl) :error-file
 			  #+(or :cmu :scl) (and *cmu-errors-to-file*
 						(component-full-pathname component
